src/bundle/Resources/public/js/scripts/sidebar/main.menu.js

```diff
@@ -0,0 +1,132 @@
+(function(global, doc, localStorage, eZ) {
```

Comment:
```suggestion
(function(global, doc, eZ, localStorage) {
```
nitpicking, but so far `eZ` (if it was used) was always third argument